### ISSUE TITLE
.wav tags with chinese characters are showed as ???

### ISSUE THREAD

**Issue by glu8716** [mpv-player/mpv#11983]
### Important Information

- mpv version: **mpv 0.35.1-dirty**
- Linux Distribution and Version: **Artix**
- Source of the mpv binary: **Artix repos**
- Window Manager and version: **Openbox 3.6.1-10**
- GPU driver and version: **Nvidia 535.86.05-2**

### Reproduction steps

- Open a .wav audio file with chinese characters in tags

### Expected behavior
- Tags should show the correct names

### Actual behavior

- Tags are displayed as ???

```
[msg@Artix [2012] 瑞鹤飞]$ mpv "02 - 长相思.wav"
 (+) Video --vid=1 [P] (mjpeg 434x600 1.000fps)
     Video --vid=2 [P] 'Cover.jpg' (mjpeg 434x600 1.000fps) (external)
 (+) Audio --aid=1 (dts 7ch 44100Hz)
File tags:
 Artist: ??
 Album: ???
 Album_Artist: 常静
 Date: 2012
 Title: ???
 Track: 2
```
### Log file

[output.txt](https://github.com/mpv-player/mpv/files/12131318/output.txt)

### Sample files

https://0x0.st/HeAK.wav

**Comment by hooke007** [comment]
Obviously it's not mpv's issue. It cannot be parsed by windows' property.
<img width=400 src="https://github.com/mpv-player/mpv/assets/41094733/145b8306-83a4-4d2d-9555-9b4ed73b254f">

**Comment by hooke007** [comment]
The problem should be using the wrong tool writing the tag info.
After editing the tag with proper software (i.e. mp3tag), it works well.
<img width=500 src="https://github.com/mpv-player/mpv/assets/41094733/84a85449-9d96-46f5-80d8-49eba3fb0f98">

**Comment by glu8716** [comment]
I'm using MusicBrainz Picard to edit tags which is to be considered "proper" I guess. Tags are recognized correctly by other players like VLC or Tauon Music Box.

![2023-07-21_22-06](https://github.com/mpv-player/mpv/assets/33501455/b7af2e74-3b53-45a9-99b1-7f5b1e6b2609)

**Comment by llyyr** [comment]
Your file is broken, it has incorrect encoding for the metadata tags, it's not even using id3v2 tags. Some software might be have some heuristic to ignore the specified encoding and try to decode it with a best guess (like VLC or some tag editor), but I don't know if ffmpeg can do that. Either way if such a feature is implemented it would have to be by ffmpeg.

This is a bug with whatever tag editor you're using, check if it only happens with wav files or also with mp3 and report the bug to them.

**Comment by glu8716** [comment]
This is only happening with .wav files. And yes, you are right. This is probably something related to ffmpeg as the details of ffprobe also can't display the tags correctly.

I've found out that the culprit are "RIFF INFO tags". By setting them to UTF-8 or by deleting them the tags are showed correctly in mpv.
![2023-07-21_22-38](https://github.com/mpv-player/mpv/assets/33501455/8b4e4545-33ad-4d39-94a2-5d8b540efd0f)

By the way, the file I provided should be using ID3v2. I tried with another tag editor (Kid3) and tags are showing as ID3v2.
![2023-07-21_22-49](https://github.com/mpv-player/mpv/assets/33501455/c32b95e4-f656-4a8c-bb10-dadc4585e764)
